server/storage/rethinkdb.go

```diff
@@ -0,0 +1,205 @@
+package storage
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/dancannon/gorethink"
+	"github.com/docker/notary/storage/rethinkdb"
+)
+
+// RDBTUFFile is a tuf file record
+type RDBTUFFile struct {
```

Comment:
totally non-blocking naming nitpick:  should this be moved into the model file, since these seem to be the data models?